src/NuGet.Core/NuGet.Common/RuntimeEnvironmentHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace NuGet.Common
+{
+    internal static class RuntimeEnvironmentHelper
```

Comment:
This exists in NuGet.Configuration, same class and methods but a slightly different impl. NuGet.Common is the right place for this I think, Config should call into this.